build-tools/packages/build-cli/package.json

```diff
@@ -158,6 +158,23 @@
 	"engines": {
 		"node": ">=14.17.0"
 	},
+	"fluidBuild": {
+		"tasks": {
+			"compile": {
+				"dependsOn": [
+					"...",
+					"build:diagrams"
+				],
+				"script": false
+			},
+			"build:diagrams": [],
+			"build:manifest": [
+				"...",
```

Comment:
Remind me what this entry did?

Reply:
It means "all the defaults from the task definition in the fluid-build config."